src/SchemaGenerator.ts

```diff
@@ -33,19 +47,33 @@ export class SchemaGenerator {
     private findRootNode(fullName: string): ts.Node {
         const typeChecker = this.program.getTypeChecker();
 
-        if (!this.allTypes) {
-            this.allTypes = new Map<string, ts.Node>();
+        if (this.prioritizedFiles.length) {
```

Comment:
This could just become 

```
for (const sourceFile of this.prioritizedFiles) {
  this.inspectNode(sourceFile, typeChecker, this.allTypes);
}
```

Same below.